### ISSUE TITLE
How to get request's status code after a do_get ?

### ISSUE THREAD

**Issue by cyril-marpaud** [jeremychone/rust-httpc-test#8]
Is there a way to get a request's status code ? Sth that would look like this:

```
let client = httpc_test::new_client(format!("http://localhost:{}", axum_test::LISTENING_PORT))?;

let status_code = client.do_get("/favicon.svg").await?.status_code().await?;
```

I can find no status-related info on the [Response](https://docs.rs/httpc-test/0.1.4/httpc_test/struct.Response.html) documentation 🤷

**Comment by jeremychone** [comment]
That was missing. thanks.

**Comment by cyril-marpaud** [comment]
Thank you for the "Rust Axum Full Course" Youtube video btw, it's absolutely awesome 🔥

**Comment by jeremychone** [comment]
Fixed as part of #9. 

@cyril-marpaud Thanks a lot for the #9 pull request. Just merged. Will be in release `0.1.5`
